vortex-array/src/array/chunked/mod.rs

```diff
@@ -46,7 +47,18 @@ impl ChunkedArray<'_> {
         let mut children = vec![chunk_ends.into_array_data()];
         children.extend(chunks.iter().map(|a| a.to_array_data()));
 
-        Self::try_from_parts(dtype, ChunkedMetadata, children.into(), StatsSet::new())
+        // NB: this reports whatever stats we can correctly deduce for the top-level logical array
+        // If we'd prefer not to report stats for non-primitive arrays we can remove this
+        let merged_stats = chunks
```

Comment:
Why do this eagerly? The chunked stats compute already does this on-read

Reply:
You're right we don't need this.